apps/mark/frontend/test/helpers/timers.ts

```diff
@@ -0,0 +1,14 @@
+import { waitFor } from '@testing-library/react';
+import { advanceTimers as advanceTimersBase } from '@votingworks/test-utils';
+import { AUTH_STATUS_POLLING_INTERVAL_MS } from '@votingworks/ui';
+
+export function advanceTimers(seconds = 0): void {
```

Comment:
Moved these few pieces out of `test/helpers/smartcards.ts` and removed the rest of `test/helpers/smartcards.ts`, which was all for voter cards